fulp_modules/features/antagonists/bloodsuckers/code/powers/_powers.dm

```diff
@@ -137,7 +140,7 @@
 	if(power_flags & BP_AM_STATIC_COOLDOWN)
 		cooldown_time = initial(cooldown_time)
 	else
-		cooldown_time = max(cooldown_time / 2, cooldown_time - (cooldown_time / 16 * (level_current-1)))
+		cooldown_time = max(initial(cooldown_time) / 2, initial(cooldown_time) - (initial(cooldown_time) / 16 * (level_current-1)))
```

Comment:
why do we need to check initial?

Reply:
otherwise cooldown_time on non static cooldowns get reduced to basically 0